frontendActions/src/main/java/com/virtuslab/gitmachete/frontend/actions/base/BaseSlideInBranchBelowAction.java

```diff
@@ -133,7 +133,7 @@ public void actionPerformed(AnActionEvent anActionEvent) {
         .map(names -> names.contains(slideInOptions.getName()))
         .getOrElse(false);
 
-    if (entryAlreadyExistsBelowGivenParent) {
+    if (entryAlreadyExistsBelowGivenParent && slideInOptions.shouldReattach()) {
```

Comment:
Unrelated bugfix, right? :thinking:

Reply:
Yup

Now, we allow sliding in a branch under the same parent but with reattaching option disabled.
Eventually, the slid in branch and its children are direct children (descendants) of the given parent.

Reply:
Maybe drop an extra comment here in the code coz it's non-trivial